opentelemetry-api/src/opentelemetry/_metrics/__init__.py

```diff
@@ -136,19 +136,35 @@ def schema_url(self):
 
     @abstractmethod
     def create_counter(self, name, unit="", description="") -> Counter:
+        """Creates a `Counter` instrument
+
+        Args:
+            name: The name of the instrument to be created
+            unit: The unit for measurements this instrument reports. For
+                example, ``By`` for bytes. UCUM units are recommended.
```

Comment:
```suggestion
                example, ``B`` for bytes. UCUM units are recommended.
```
As per [this](https://en.wikipedia.org/wiki/Byte#Unit_symbol).